node/querystring.ts

```diff
@@ -433,7 +433,7 @@ const unhexTable = new Int8Array([
  * A safe fast alternative to decodeURIComponent
  */
 export function unescapeBuffer(s: string, decodeSpaces = false): Buffer {
-  const out = new Buffer(s.length);
+  const out = Buffer.from(s);
```

Comment:
This type of Buffer.from call seems causing TypeError.

```
> Buffer.from(10)
Uncaught:
TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received type number (10)
```

Reply:
@kt3k that matches the behavior in Node.

Reply:
I replaced `Buffer.from(<integer>)` with `Buffer.alloc(<integer>)`.